ReactComponents/NetworkImage.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-import React, {
+import React from 'react';
+
+import {
   Component,
```

Comment:
Moved this into React, but thinking we actually don't need (refs https://github.com/DoSomething/LetsDoThis-iOS/pull/1027/files#r64401642)